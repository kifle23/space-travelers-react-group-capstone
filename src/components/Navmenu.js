import React from 'react';
import { NavLink } from 'react-router-dom';
import logo from '../logo.png';

const Navmenu = () => (
  <header>
    <nav className="container d-flex-space">
      <div className="d-flex-space g-10">
        <img src={logo} alt="logo" className="logo-img" />
        <h2 className="logo-title">Space Traveler&#39;s Hub</h2>
      </div>
      <ul className="d-flex-space g-10">
        <li><NavLink to="/">Rockets</NavLink></li>
        <li><NavLink to="/missions">Missions</NavLink></li>
        |
        <li><NavLink to="/profile">My Profile</NavLink></li>
      </ul>
    </nav>
  </header>
);

export default Navmenu;
