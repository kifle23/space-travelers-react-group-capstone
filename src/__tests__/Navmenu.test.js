import { BrowserRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import Navmenu from '../components/Navmenu';

it('renders correctly', () => {
  const tree = renderer
    .create(
      <BrowserRouter>
        <Navmenu />
      </BrowserRouter>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
