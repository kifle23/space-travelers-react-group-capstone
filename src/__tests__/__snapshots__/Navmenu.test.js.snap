// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<header>
  <nav
    className="container d-flex-space"
  >
    <div
      className="d-flex-space g-10"
    >
      <img
        alt="logo"
        className="logo-img"
        src="logo.png"
      />
      <h2
        className="logo-title"
      >
        Space Traveler's Hub
      </h2>
    </div>
    <ul
      className="d-flex-space g-10"
    >
      <li>
        <a
          aria-current="page"
          className="active"
          href="/"
          onClick={[Function]}
        >
          Rockets
        </a>
      </li>
      <li>
        <a
          className=""
          href="/missions"
          onClick={[Function]}
        >
          Missions
        </a>
      </li>
      |
      <li>
        <a
          className=""
          href="/profile"
          onClick={[Function]}
        >
          My Profile
        </a>
      </li>
    </ul>
  </nav>
</header>
`;
